src/SchemaRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
```

Comment:
please lock version for all code produced by ourselves

Reply:
Shall we add it to the linter config as well?

Reply:
Yes we can!